task/09-functions-n-closures-tasks.js

```diff
@@ -84,10 +94,16 @@ function getPolynom() {
  *   memoizer() => the same random number  (next run, returns the previous cached result)
  */
 function memoize(func) {
-    throw new Error('Not implemented');
+  let memo = {};
+  return function () {
+    const args = JSON.stringify(arguments);
+    if (memo[args] === undefined) {
```

Comment:
It'd be better to use `in` operator since `func.apply(this, arguments)` in theory can return `underfiend`